pkgs/tools/networking/ssl-proxy/default.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, buildGo118Module, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub}:
```

Comment:
Shouldn't there be a space between `fetchFromGithub` and `}`? Sorry for being picky. :confused: